src/sysen3/papsav.2

```diff
@@ -0,0 +1,385 @@
+	TITLE PAPSAV
+
+	; EJS (2018-12-20): Sourcee reconstructed CHANNA;RAKASH PAPSAV binary.
+        ; This reconstructed version uses macros from SYSENG; CALRET > as the original
+	; source did. The original PAPSAV shared some code with SYSEN1;SYSMSG > (which
+	; also inserted SYSENG; CALRET >). Where applicable, the source from SYSMSG was
+	; used in this reconstruction.
+
+.INSRT SYSENG;CALRET >
+
+t1=1
+t=2
+tt=3
+a=4
+.prsva=4
+b=5
+c=6
+f=7
+pt=10
+ch=11
+count=12
+.prsvz=13
+.ap.=14
+.pf.=15
+.cf.=16
```

Comment:
Some of these are defined in CALRET, so they don't need to be defined in PAPSAV.

Reply:
Right. Forgot to remove them.  I'll fix.